content/docs/1.9/getting-started.md

```diff
@@ -119,8 +119,7 @@ Then navigate to `http://localhost:8080`.
 
 ## Migrating from Zipkin
 
-Collector service exposes Zipkin compatible REST API `/api/v1/spans` and `/api/v2/spans` for both
```

Comment:
why 1.9 and not 1.10 ?

Reply:
Actually, this is not changed much. Just made it one line instead of two lines. The content is the same.

Reply:
I am saying 1.9 is already N-1 release, we have 1.1o now: https://www.jaegertracing.io/docs/1.10/getting-started/#migrating-from-zipkin

Reply:
I see. I've changed my updates to alter v1.10 docs.